migrations/20210927094004_payment/migration.sql

```diff
@@ -0,0 +1,283 @@
+/*
+  Warnings:
+
+  - The values [organisation] on the enum `beneficiary_type` will be removed. If these variants are still used in the database, this will fail.
+  - The `currency` column on the `campaigns` table would be dropped and recreated. This will lead to data loss if there is data in the column.
+  - A unique constraint covering the columns `[personal_number]` on the table `people` will be added. If there are existing duplicate values, this will fail.
+
+*/
+-- CreateEnum
+CREATE TYPE "currency" AS ENUM ('BGN', 'EUR', 'USD');
+
+-- CreateEnum
+CREATE TYPE "transaction_type" AS ENUM ('donation');
+
+-- CreateEnum
+CREATE TYPE "transfer_type" AS ENUM ('withdraw', 'relocation');
+
+-- CreateEnum
+CREATE TYPE "expense_type" AS ENUM ('none', 'internal', 'operating', 'administrative', 'medical', 'services', 'groceries', 'transport', 'accommodation', 'shipping', 'utility', 'rental', 'legal', 'bank', 'advertising', 'other');
+
+-- CreateEnum
+CREATE TYPE "payment_provider" AS ENUM ('none', 'stripe', 'paypal', 'epay', 'bank', 'cash');
+
+-- CreateEnum
+CREATE TYPE "subscription_status" AS ENUM ('trialing', 'active', 'canceled', 'incomplete', 'incompleteExpired', 'pastDue', 'unpaid');
+
+-- CreateEnum
+CREATE TYPE "vault_type" AS ENUM ('personal', 'campaign');
+
+-- CreateEnum
+CREATE TYPE "document_type" AS ENUM ('invoice', 'receipt', 'medical_record', 'other');
+
+-- CreateEnum
+CREATE TYPE "transaction_status" AS ENUM ('initial', 'invalid', 'incomplete', 'declined', 'waiting', 'cancelled', 'succeeded', 'deleted', 'refund', 'paymentRequested');
+
+-- CreateEnum
+CREATE TYPE "payout_type" AS ENUM ('cash', 'bank');
+
+-- CreateEnum
+CREATE TYPE "payout_status" AS ENUM ('initial', 'invalid', 'incomplete', 'declined', 'cancelled', 'succeeded');
+
+-- CreateEnum
+CREATE TYPE "account_holder_type" AS ENUM ('individual', 'company');
+
+-- CreateEnum
+CREATE TYPE "bank_account_status" AS ENUM ('new', 'validated', 'verified', 'verification_failed', 'errored');
```

Comment:
could you tell what is the difference between validated and verified?
errored sounds quite strange - a bank account may be invalid, but "errored" looks like a "failed" which is more appropriate for a transaction

Reply:
aah poor naming spilling from Stripe into our domain ;))) ok fine then, will be good to add comment in the schema where these are coming from...